int64 update_time
int16[] data
---
string result
